lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -568,9 +568,8 @@ MStatus MayaUsdProxyShapeBase::computeInStageDataCached(MDataBlock& dataBlock)
                 if (SdfLayerRefPtr rootLayer = SdfLayer::FindOrOpen(fileString)) {
                     SdfLayerRefPtr sessionLayer = computeSessionLayer(dataBlock);
 
-                    bool targetSession
-                        = MGlobal::optionVarIntValue(UsdMayaUtil::convert(
-                              MayaUsdOptionVars->mayaUsd_ProxyTargetsSessionLayerOnOpen))
+                    bool targetSession = MGlobal::optionVarIntValue(UsdMayaUtil::convert(
+                                             MayaUsdOptionVars->ProxyTargetsSessionLayerOnOpen))
                         == 1;
```

Comment:
TBH, both styles forced by clang-format look a bit weird. Maybe adding a parenthesis pair to the right hand side of the assignment could make it better?

Reply:
Or declare another variable to hold the return value of UsdMayaUtil::convert()?

Reply:
Addressed in 991bdb68dc8ca260b2174f33b705f8a0675a4b17.